acousticbrainz/data/dataset.py

```diff
@@ -129,7 +131,7 @@ def get(id):
     try:
         connection = psycopg2.connect(current_app.config["PG_CONNECT"])
         cursor = connection.cursor()
```

Comment:
Huh, I didn't even know something like that existed. I will try to integrate it there.